apps/api/start.sh

```
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+echo "Fetching secrets from Infisical..."
+export INFISICAL_TOKEN=$(infisical login --method=universal-auth --client-id=$INFISICAL_CLIENT_ID --client-secret=$INFISICAL_CLIENT_SECRET --plain --silent)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)